packages/x-date-pickers/src/internals/hooks/useDefaultReduceAnimations.ts

```diff
@@ -2,8 +2,15 @@ import useMediaQuery from '@mui/material/useMediaQuery';
 
 const PREFERS_REDUCED_MOTION = '@media (prefers-reduced-motion: reduce)';
 
+// detect if user agent has Android version < 10 or iOS version < 13
+const mobileVersionMatches =
+  typeof navigator !== 'undefined' && navigator.userAgent.match(/android\s(\d+)|OS\s(\d+)/i);
+const androidVersion =
+  mobileVersionMatches && mobileVersionMatches[1] ? parseInt(mobileVersionMatches[1], 10) : null;
+const iOSVersion =
```

Comment:
This might not even be needed, such a tiny audience: https://caniuse.com/usage-table but no harm.

Reply:
On Android you can be on an oldish OS version, but use the latest chrome browser.
As far as I can find iOS aligns Safari browser versions and their updates with the system OS version... 🤔 
Do you think that it then would make sense to adjust the rule to `< 14` or maybe even `< 15` instead of `< 13`?

Reply:
You can use new iOS version on old iOS devices, I believe. I think we are good right now.

Reply:
@oliviertassinari 
AFAIK, iPhones do get updates for quite a few years, 6 at minimum usually, but from what I've read, the Safari update is bundled with the OS update, hence, they will basically always be the same as the phone OS version. 🤔